vue.config.js

```diff
@@ -1,15 +1,10 @@
 module.exports = {
-    pluginOptions: {
-      electronBuilder: {
-        builderOptions: {
-
+  pluginOptions: {
+    electronBuilder: {
+      builderOptions: {
         // options placed here will be merged with default configuration and passed to electron-builder
-        extraResources: [
-            "example-tds",
-            "./node_modules/virtual-thing"
-            ]
-
-        }
-      }
-    }
-  }
\ No newline at end of file
+        extraResources: ["example-tds", "./node_modules/virtual-thing"],
```

Comment:
Is this intentional?

Reply:
That I didn't change, my vscode auto-formatted the file so it shows-up as a change.

Reply:
Could you use editorconfig which works by creating a conf file and a vs-code extension that detects it and uses the indentation preferences

Reply:
Sure, I got this working now. But @egekorkan or @neniman could you please send me your .editorconfig file, since the auto-generated one with the plugin still doesn't match wade's config. Thanks!